homeassistant/components/tibber/sensor.py

```diff
@@ -335,15 +335,19 @@ def __init__(
             "accumulated production",
             "accumulated cost",
         ]:
-            self._attr_last_reset = dt_util.now().replace(
-                hour=0, minute=0, second=0, microsecond=0
+            self._attr_last_reset = (
+                dt_util.now()
+                .replace(hour=0, minute=0, second=0, microsecond=0)
+                .astimezone(timezone.utc)
             )
         elif self._sensor_name in [
             "accumulated consumption last hour",
             "accumulated production last hour",
         ]:
-            self._attr_last_reset = dt_util.now().replace(
-                minute=0, second=0, microsecond=0
+            self._attr_last_reset = (
+                dt_util.now()
```

Comment:
Just use `dt_util.utcnow` instead.

Reply:
I think that will be incorrect in Nepal :) GMT+5:45
(But Tibber is not available in Nepal yet :) )